packages/functional-tests/tests/signin/signIn.spec.ts

```diff
@@ -39,15 +33,14 @@ test.describe('severity-2 #smoke', () => {
     test('signin with email with leading/trailing whitespace on the email', async ({
       target,
       page,
-      credentials,
       pages: { login },
+      accountManager,
     }) => {
-      const emailWithleadingSpace = '   ' + credentials.email;
+      const { email, password } = accountManager.create();
```

Comment:
`.create()` or `.createSync()`?

Reply:
`.create()`. It might be confusing because it never took emailOptions into consideration.